cpp/src/arrow/array/array_list_test.cc

```diff
@@ -197,6 +197,37 @@ class TestListArray : public ::testing::Test {
     EXPECT_FALSE(left->Slice(offset)->Equals(right->Slice(offset)));
   }
 
+  void TestFromArraysWithNullBitMap() {
+    std::shared_ptr<Array> offsets_w_nulls, offsets_wo_nulls, values;
+
+    std::vector<offset_type> offsets = {0, 1, 1, 3, 4};
+
+    std::vector<bool> offsets_w_nulls_flags = {true, false, true, true, true};
+    std::vector<bool> offsets_wo_nulls_flags = {true, true, true, true, true};
+
+    std::vector<int32_t> null_bitmap_values = {1, 0, 1, 1};
```

Comment:
I guess this is ok as an arbitrary null bitmap buffer, but for a 4-entries array this will be all ones, right?

Reply:
Just removed it, realized I could just re-use the `expected->null_bitmap()`